examples/waspello/ext/MainPage.js

```diff
@@ -169,12 +169,16 @@ const Lists = ({ lists, listIdToCardsMap }) => {
 
 const List = ({ list, index, cards }) => {
   const [isPopoverOpen, setIsPopoverOpen] = useState(false)
+  const [showHeaderTarget, setShowHeaderTarget] = useState(true);
```

Comment:
```suggestion
  const [isHeaderTargetShown, setIsHeaderTargetShown] = useState(true);
```
We usually start boolean values with `is`, so tried to rewrite it to follow this rule.

Reply:
Done